src/app/(main)/orders/page.tsx

```diff
@@ -0,0 +1,7 @@
+import dynamic from "next/dynamic";
+
+export default function OrderPage() {
+	const OrderList = dynamic(() => import("../../../ui/components/OrderList"), { ssr: false });
```

Comment:
I think this should be outside the component body?
```suggestion
const OrderList = dynamic(() => import("../../../ui/components/OrderList").then((m) => m.OrderList), { ssr: false });

export default function OrderPage() {
```

Reply:
@mmiszy named export doesn't work unfortunately

Reply:
It does work, we're already using in other places.

Reply:
@mmiszy can you check on your side?

![CleanShot 2023-11-11 at 14 00 10@2x](https://github.com/saleor/storefront/assets/200613/765611e0-4846-4077-8899-f6173bc3e2ed)

(with re-created `.next/`)

Reply:
Indeed there's a bug in Next.js :D Adding "use client" to the `page.tsx` fixes it.

Reply:
:D 

Yeah, but I'd prefer not to set the entire page as `client`. Let's keep it as `default` for now with an eslint exception, there's an open bug report in the Next.js repo, let's track it and fix it once it's done upstream. WDYT?

Reply:
This whole page is client anyway because you're only rendering a single dynamic component.